src/routes/people/+page.svelte

```diff
@@ -1,4 +1,4 @@
-<script>
+<script lang="ts">
```

Comment:
No need for TypeScript here. As far as I can tell, the syntax here is all sufficiently JavaScript.
```suggestion
<script>
```